docs/src/systems/weakly_compressible_sph.md

```diff
@@ -152,8 +152,74 @@ as explained in [Sun2018](@cite Sun2018) on page 29, right above Equation 9.
 The ``\delta``-SPH method (WCSPH with density diffusion) together with this formulation
 of PST is commonly referred to as ``\delta^+``-SPH.
 
-The Particle Shifting Technique can be applied in form
-of the [`ParticleShiftingCallback`](@ref).
+To apply particle shifting, use the keyword argument `shifting_technique` in the constructor
+of a system that supports it.
+
+
+## [Transport Velocity Formulation (TVF)](@id transport_velocity_formulation)
+
+An alternative formulation is the so-called Transport Velocity Formulation (TVF)
+by [Adami (2013)](@cite Adami2013).
+[Ramachandran (2019)](@cite Ramachandran2019) applied the TVF also for the [EDAC](@ref edac)
+scheme.
```

Comment:
Note that I just copied this section over from EDAC and changed this first paragraph. The rest of the section is unchanged, except for changes like "right hand side" to "right-hand side" and "time-step" to "time step".